packages/app/index.node.ts

```diff
@@ -18,8 +18,11 @@
 import { FirebaseNamespace } from '@firebase/app-types';
 import { _FirebaseNamespace } from '@firebase/app-types/private';
 import { createFirebaseNamespace } from './src/firebaseNamespace';
-import Storage from 'dom-storage';
-import { XMLHttpRequest } from 'xmlhttprequest';
+// Node specific packages.
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const Storage = require('dom-storage');
```

Comment:
`dom-storage` and `xmlhttprequest` don't have types so the type of `Storage` and `XMLHttpRequest` is implicitly `any`.  Maybe it would be simpler to just `@ts-ignore` and explain it is a node package.